plasma_framework/contracts/src/framework/registries/ExitGameRegistry.sol

```diff
@@ -1,18 +1,29 @@
 pragma solidity ^0.5.0;
 
 import "../utils/Operated.sol";
+import "../utils/Quarantine.sol";
 
 contract ExitGameRegistry is Operated {
     mapping(uint256 => address) private _exitGames;
     mapping(address => uint256) private _exitGameToTxType;
+    using Quarantine for Quarantine.Data;
+    Quarantine.Data internal _quarantine;
 
     event ExitGameRegistered(
         uint256 txType,
         address exitGameAddress
     );
 
-    modifier onlyFromExitGame() {
+    constructor (uint256 _minExitPeriod, uint256 _initialImmuneExitGames)
+        public
+    {
+        _quarantine.quarantinePeriod = _minExitPeriod;
```

Comment:
different from Vault, the `quarantinePeriod` should be `2 * _minExitPeriod` I believe.